src/FilenameParsing/FileIDHelperResolutionStrategy.php

```diff
@@ -353,6 +373,12 @@ public function setVersionedStage($versionedStage)
     }
 
 
+    /**
+     * todo: needs documenting
```

Comment:
Document

Reply:
Those methods that didn't have a PHPDoc block are implemented from an interface that has the doc block. The interface doc block are inherited.